quinn/examples/common/mod.rs

```diff
@@ -0,0 +1,77 @@
+//! Commonly used code in most examples.
+
+use quinn::{
+    Certificate, CertificateChain, ClientConfig, ClientConfigBuilder, Driver, Endpoint, Incoming,
+    PrivateKey, ServerConfig, ServerConfigBuilder, TransportConfig,
+};
+use std::error::Error;
+use std::net::ToSocketAddrs;
+use std::sync::Arc;
+
+/// Returns QUIC endpoint builder configured to build a client endpoint.
```

Comment:
This does not return a builder.

```suggestion
/// Constructs a QUIC endpoint configured for use as a client only.
```